test/test/bootstrap.js

```diff
@@ -43,7 +43,7 @@ describe('Bridge', function () {
     });
 
     it('should transfer ETH from Ethereum to Substrate (incentivized channel)', async function () {
-      const amount = BigNumber(Web3.utils.toWei('1', "ether"));
+      const amount = BigNumber(Web3.utils.toWei('0.1', "ether"));
```

Comment:
`0.1` is still quite a lot for ropsten. May want to reduce it further to `0.001`.